{
  "Header": {
    "Original": "CVE-2022-4645 (LibTIFF 4.4.0 has an out-of-bounds read in tiffcp in tools/tiffcp.c:94 ...)",
    "ID": "CVE-2022-4645",
    "Description": "(LibTIFF 4.4.0 has an out-of-bounds read in tiffcp in tools/tiffcp.c:94 ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-5333-1}",
      "Type": "xref",
      "Bugs": [
        "DSA-5333-1"
      ]
    },
    {
      "Original": "- tiff 4.4.0-5",
      "Type": "package",
      "Package": "tiff",
      "Kind": "fixed",
      "Version": "4.4.0-5"
    },
    {
      "Original": "[buster] - tiff \u003cpostponed\u003e (Minor issue, OOB read / DoS)",
      "Type": "package",
      "Release": "buster",
      "Package": "tiff",
      "Kind": "postponed",
      "Description": "Minor issue, OOB read / DoS"
    },
    {
      "Original": "NOTE: https://gitlab.com/libtiff/libtiff/-/issues/277",
      "Type": "NOTE",
      "Description": "https://gitlab.com/libtiff/libtiff/-/issues/277"
    },
    {
      "Original": "NOTE: https://gitlab.com/libtiff/libtiff/-/commit/e813112545942107551433d61afd16ac094ff246",
      "Type": "NOTE",
      "Description": "https://gitlab.com/libtiff/libtiff/-/commit/e813112545942107551433d61afd16ac094ff246"
    },
    {
      "Original": "NOTE: The fix causes CVE-2023-2908.",
      "Type": "NOTE",
      "Description": "The fix causes CVE-2023-2908."
    }
  ]
}